website/docs/r/database_secret_backend_static_role.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: "vault"
+page_title: "Vault: vault_database_secret_backend_static_role resource"
+sidebar_current: "docs-vault-resource-database-secret-backend-static-role"
+description: |-
+  Configures a database secret backend static role for Vault.
+---
+
+# vault\_database\_secret\_backend\_static\_role
+
+Creates a Database Secret Backend static role in Vault. Database secret backend
+static roles can be used to manage 1-to-1 mapping of a Vault Role to a user in a
+database for the database.
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
```

Comment:
It's awesome you point this out. Do you think we should mark any of the new schema fields Sensitive to protect against this further?

Reply:
I'm removing this because I after second look it appears to me that there are no secret schema fields.